app/controllers/maintenance_tasks/tasks_controller.rb

```diff
@@ -19,12 +19,17 @@ def index
     # Shows running and completed instances of the Task.
     def show
       @task = TaskData.find(params.fetch(:id))
+      @last_run = @task.last_run
+      set_refresh if @last_run&.active?
```

Comment:
We already have memoization in `TaskData#last_run`, I don't think we need a instance variable for the last run.